docs/REST API/rest-api.md

```diff
@@ -3986,18 +3986,28 @@ List all configuration profiles for macOS hosts enrolled to Fleet's MDM that are
       "profile_id": 1337,
       "team_id": 0,
       "name": "Example profile",
+      "platform": "macOS",
       "identifier": "com.example.profile",
       "created_at": "2023-03-31T00:00:00Z",
       "updated_at": "2023-03-31T00:00:00Z",
       "checksum": "dGVzdAo="
+    },
+    {
+      "profile_id": 1338,
+      "team_id": 0,
+      "name": "Example Windows profile",
+      "platform": "Windows",
```

Comment:
```suggestion
      "platform": "windows",
```

In the API, we use lowercase `windows`. Also more of a historical thing we should try to stay consistent with.